api/sample.env

```diff
@@ -1,7 +1,7 @@
-DB_HOST=<host_address_and_port>
+PG_DB_IP=<host_address_and_port>

```

Comment:
We should add the new environment variables for the API to this sample.env. I know we have the global one in the parent directory but it would be nice to update this one as well to show which ones are required for the API on its own.

Reply:
Addressed